docs/quickstart-config.yaml

```diff
@@ -22,11 +22,7 @@ dataStore:
   driver: org.postgresql.Driver
 
 clusterStatsConfiguration:
-  useApi: true
-
-backendState:
-  username: gateway
-  ssl: false
+  clusterStatsMonitorType: INFO_API
```

Comment:
`clusterStatsConfiguration.clusterStatsMonitorType` seems redundant. How about renaming to `monitorType`?